src/modelbench/benchmark_runner.py

```diff
@@ -336,7 +353,9 @@ def handle_item(self, item: TestRunItem) -> TestRunItem:
         return item
 
     def collect_annotations(self, item):
-        for annotator_key, annotator in item.test.get_annotators().items():
+        # TODO: Parallelize annotators
```

Comment:
This is already parallel if the `thread_count` parameter is greater than 1. If a particular run needs faster annotation, easiest just to increase that number. It would be possible to also split this into 3 stages (an assigner stage, a single-annotation worker stage, and a collection/measurement stage), but if you'd like to do that please make an empirical case that it'll improve the experience for some user, as my intuition is that it won't make much of a practical difference.

Reply:
For example, the ensemble annotator from ws3 uses annotators hosted on different platforms (e.g. together, huggingface, etc..). The sequential annotation process introduces an avoidable bottleneck when running a diverse annotator set. We could get a significant speed boost from parallelizing the annotation stage. I don't think it's super high priority, just something to think about in the future.

Reply:
I understand that multiple annotators will sometimes be involved, and that splitting the annotation would have a theoretical benefit,  but I'm not convinced there is a significant speed boost to be had for real world runs. So please make an empirical case, one with actual measured numbers, to demonstrate that the additional complexity is justified by some user's improved experience.

Reply:
I don't have those numbers so I will delete the todo.